src/components/person/person.component.ts

```diff
@@ -0,0 +1,31 @@
+import { PersonService} from './person.service'
+import { Man } from './types';
+import { MansData } from './types';
+export class Person {
+  public firstName: string;
+  public lastName: string;
+  private currentMan: number;
+  public message: string;
+  public showTasks : boolean;
+  public listSurnames: string[];
+  constructor(private service: PersonService) {
+  }
+
+  $onInit(){
+    this.showTasks = false;
+  }
+
+  public getMessage() : void {
+    this.message= this.service.getMessage( this.service.mansData[ this.currentMan ] );
```

Comment:
this.service.mansData[ this.currentMan ] => this.service.mansData[this.currentMan]